test/RelayHubValidator.test.ts

```diff
@@ -0,0 +1,72 @@
+require('source-map-support').install({ errorFormatterForce: true })
+
+const TestRelayHubValidator = artifacts.require('TestRelayHubValidator')
+
+contract('RelayHubValidator', ([from, senderAddress, target, paymaster, relayWorker, forwarder]) => {
+  let validator: any
+  before(async () => {
+    validator = await TestRelayHubValidator.new()
+  })
+
+  it('#len1 should return dynamic structure size', async () => {
+    const len1 = async function (size: number): Promise<number> { return validator.len1('0x' + '11'.repeat(size)).then((x: any) => x.toNumber()) }
```

Comment:
please give meaningful names to lambda functions

Reply:
localWrapperForSolidityLen1ToBeEasilyCalledFromAssert ?